src/e3/log.py

```diff
@@ -334,13 +334,7 @@ def add_logging_argument_group(
         "--loglevel",
         default=default_level,
         help="set the console log level",
-        choices={
-            "DEBUG": logging.DEBUG,
-            "INFO": logging.INFO,
-            "WARNING": logging.WARNING,
-            "ERROR": logging.ERROR,
-            "CRITICAL": logging.CRITICAL,
-        },
+        choices=["DEBUG", "INFO", "WARNING", "ERROR", "CRITICAL"],
```

Comment:
That will be hard to maintain, I would put a LEVELS constant and use choices=list(LEVELS.keys())